### PR TITLE
Feat: Contianer autoRestart

### PR DESCRIPTION
**Pull request by Meldiron**
For execution containers its useful to have ability to autoRestart after a crash. I added new param to support exactly that.

### CODE REVIEW

File: app/http.php [hunk 1]
```diff
@@ -411,11 +411,12 @@ function removeAllRuntimes(Table $activeRuntimes, Orchestration $orchestration):
     ->param('cpus', 1, new Integer(), 'Container CPU.', true)
     ->param('memory', 512, new Integer(), 'Comtainer RAM memory.', true)
     ->param('version', 'v4', new WhiteList(['v2', 'v4']), 'Runtime Open Runtime version.', true)
+    ->param('autoRestart', false, new Boolean(), 'Runtime Open Runtime version.', true)
```
Comment (christyjacob4) [inline app/http.php:414]: Why dont we use `restartPolicy` like we did for orchestration ? 
also description needs to be updated

**Comment by Meldiron** [comment]
test s passing fine locally, but fails 5 times on ci/cd. Increastign timeout, CPU probably got slower. or dart got stronger.

CleanShot 2024-07-30 at 10 48 02@2x

File: app/http.php [hunk 2]
```diff
@@ -411,11 +411,12 @@ function removeAllRuntimes(Table $activeRuntimes, Orchestration $orchestration):
     ->param('cpus', 1, new Integer(), 'Container CPU.', true)
     ->param('memory', 512, new Integer(), 'Comtainer RAM memory.', true)
     ->param('version', 'v4', new WhiteList(['v2', 'v4']), 'Runtime Open Runtime version.', true)
+    ->param('restartPolicy', DockerAPI::RESTART_NO, new WhiteList([DockerAPI::RESTART_NO, DockerAPI::RESTART_ALWAYS, DockerAPI::RESTART_ON_FAILURE, DockerAPI::RESTART_UNLESS_STOPPED], true), 'Define restart policy once exit code is returned by command. Default value is "no". Possible values are "no", "always", "on-failure", "unless-stopped".', true)
```
Comment (christyjacob4) [inline app/http.php:414]:
```suggestion
    ->param('restartPolicy', DockerAPI::RESTART_NO, new WhiteList([DockerAPI::RESTART_NO, DockerAPI::RESTART_ALWAYS, DockerAPI::RESTART_ON_FAILURE, DockerAPI::RESTART_UNLESS_STOPPED], true), 'Define restart policy for the runtime once an exit code is returned. Default value is "no". Possible values are "no", "always", "on-failure", "unless-stopped".', true)
```

maybe we should use `explode()` and use the constants itself

**Review by christyjacob4 — APPROVED** [review]
Just one more comment and we can merge

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**